workbench-core/example/infrastructure/integration-tests/support/utils/regExpressions.ts

```diff
@@ -2,8 +2,11 @@
  *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
  *  SPDX-License-Identifier: Apache-2.0
  */
-import { resourceTypeToKey, uuidWithLowercasePrefixRegExp } from '@aws/workbench-core-base';
+import { uuidWithLowercasePrefixRegExp } from '@aws/workbench-core-base';
+import { endPointPrefix, dataSetPrefix } from '@aws/workbench-core-example-app/lib/configs/constants';
 
-export const envUuidRegExp: RegExp = uuidWithLowercasePrefixRegExp(resourceTypeToKey.environment);
```

Comment:
why is this being removed?

Reply:
We arent integrating environments into the example app so its not necessary

Reply:
Was originally here as a relic of the copy of how SWB set up their integ test infrastructure